libraries/AP_Motors/AP_MotorsHeli_RSC.cpp

```diff
@@ -392,10 +419,7 @@ void AP_MotorsHeli_RSC::update_rotor_runup(float dt)
             _rotor_runup_output = _rotor_ramp_output;
         }
     }else{
-        _rotor_runup_output -= runup_increment;
-        if (_rotor_runup_output < _rotor_ramp_output) {
-            _rotor_runup_output = _rotor_ramp_output;
-        }
+        _rotor_runup_output = _rotor_ramp_output;
```

Comment:
This is not allowed.  the Run up timer must count down to allow rotor rundown on the ground to judge when it is safe to disarm.